scraper/docker-splash.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
```

Comment:
Dit wordt nu niet gebruikt, maar is nodig als blijkt dat we javascript code uit moeten voeren om de juiste HTML terug te krijgen